lib/OpenQA/Command.pm

```diff
@@ -101,8 +92,7 @@ sub run {
     return $self->command(@args);
 }
 
-sub url_for {
-    my ($self, $path) = @_;
+sub url_for ($self, $path) {
```

Comment:
hm, `url_for` https://docs.mojolicious.org/Mojolicious/Controller#url_for might be called with more arguments.
Let's see if tests are failing.

Reply:
I just tested it locally and replaced
`my $url = $self->url_for($path);`
with
`my $url = $self->url_for($path, { x => 23 });`
in `lib/OpenQA/CLI/api.pm` and calling `perl script/openqa-cli api jobs/1 --o3 ` and it died.
So better add a `@` here.

Reply:
There's no relation to the Mojolicious `url_for` method though.

Reply:
oh I see. then ignore my comment

Reply:
Bit of a confusing naming choice here, no idea why i did that. 🤷

Reply:
Yeah, my local tests didnt fail but I wanted to see if tests fail here